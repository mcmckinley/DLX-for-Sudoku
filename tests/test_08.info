No Solution 9x9