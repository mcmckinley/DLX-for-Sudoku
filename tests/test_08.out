The input sudoku has no valid solution.
